README.md

```diff
@@ -66,7 +66,7 @@ Options:
               # The consumer application version
   -h, [--branch=BRANCH]
               # Repository branch of the consumer version
-  -r, [--auto-detect-version-properties], [--no-auto-detect-version-properties]
+  -r, [--auto-detect-version-properties], [--no-auto-detect-version-properties], [--skip-auto-detect-version-properties]
```

Comment:
What's going on with all of these `--skip-..` variants?

Reply:
I just ran a command from README to re-generate docs, so I guess someone added these flags but never re-generated the docs

Reply:
Thanks Stan. @YOU54F @pahnin any ideas as to why these `--skip` things are appearing? It's not entirely clear how they're different to the `--no-...` options.

Reply:
Looks like it came from this commit, (documenting of the option), which was released in thor 1.3.1. Downgrading thor to 1.3.0 and running `ruby script/update-cli-usage-in-readme.rb` will generate the readme with the `--skip*` opts

https://github.com/rails/thor/commit/686bcd57aa1ac7d127012b3ad443695656eb97f7

The option itself has been there since 2009 - [commit](https://github.com/rails/thor/commit/392cd4a1c6554f2ef693464c1781fe54556b2e7a)

effectively `--no-foo` is the same as `--skip-foo`

Reply:
You can rebase against master now as I've performed that readme update in a separate PR

https://github.com/pact-foundation/pact_broker-client/pull/176